modules/projects/projectList.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+if (isset($_POST["projectId"])) {
+    $projectId = mysqli_real_escape_string($db, $_POST["projectId"]);
+
+    $deleteProject = "DELETE FROM projects WHERE id='" . $projectId . "'";
+    $deleteProjectResult = mysqli_query($db, $deleteProject);
+    $deleteProjectResult;
```

Comment:
No need to assign to a variable, this is enough:
`mysqli_query($db, $deleteProject);`